analyses/data-pre-release-qc/01-data-harmonization-qc.Rmd

```diff
@@ -0,0 +1,752 @@
+                  experimental_strategy == "Targeted Sequencing", 
+                !is.na(RNA_library), 
+                sample_type == "Tumor",
+                !cohort %in% c("DGD", "TCGA")) %>%
+  dplyr::pull(Kids_First_Biospecimen_ID)
+missing_ids <- setdiff(hist_ids, biospecimen_ids)
+missing_hist_ids <- setdiff(biospecimen_ids, hist_ids)
+if (length(missing_ids) > 0) {
+  output_file <- "histologies-samples-missing-in-fusion-annoFuse.tsv"
+  missing_ids_df <- missing_biospecimen_table(hist_df, missing_ids, output_file)
+  missing_ids_df
+}
+print(paste("RNA biospecimen in histolgies missing in fusion-annoFuse.tsv.gz = ", length(missing_ids)))
+if (length(missing_hist_ids) > 0) {
+  output_file <- "fusion-annoFuse-samples-missing-in-histologies.tsv"
+  missing_hist_ids_df <- tibble::tibble(Kids_First_Biospecimen_ID = missing_hist_ids)
+  missing_hist_ids_df %>% readr::write_tsv(file.path(results_dir, output_file))
+  missing_hist_ids_df
+}
+print(paste("RNA biospecimen in fusion-annoFuse.tsv.gz missing in histolgies = ", length(missing_hist_ids)))
+```
+
+- Arriba fusion file (fusion-arriba.tsv.gz)
+```{r}
+# check that all biospecimen IDs are in the file
+biospecimen_ids <- readr::read_tsv(file.path(data_dir, "fusion-arriba.tsv.gz")) %>%
+  dplyr::pull(tumor_id) %>%
+  unique()
+hist_ids <- hist_df %>%
+  dplyr::filter(experimental_strategy == "RNA-Seq" |
+                  experimental_strategy == "Targeted Sequencing", 
+                !is.na(RNA_library), 
+                sample_type == "Tumor",
+                !cohort %in% c("DGD", "TCGA")) %>%
+  dplyr::pull(Kids_First_Biospecimen_ID)
+missing_ids <- setdiff(hist_ids, biospecimen_ids)
+missing_hist_ids <- setdiff(biospecimen_ids, hist_ids)
+if (length(missing_ids) > 0) {
+  output_file <- "histologies-samples-missing-in-fusion-annoFuse.tsv"
+  missing_ids_df <- missing_biospecimen_table(hist_df, missing_ids, output_file)
+  missing_ids_df
+}
+print(paste("RNA biospecimen in histolgies missing in fusion-arriba.tsv.gz = ", length(missing_ids)))
+if (length(missing_hist_ids) > 0) {
+  output_file <- "fusion-annoFuse-samples-missing-in-histologies.tsv"
+  missing_hist_ids_df <- tibble::tibble(Kids_First_Biospecimen_ID = missing_hist_ids)
+  missing_hist_ids_df %>% readr::write_tsv(file.path(results_dir, output_file))
+  missing_hist_ids_df
+}
+print(paste("RNA biospecimen in fusion-arriba.tsv.gz missing in histolgies = ", length(missing_hist_ids)))
+```
+
+- STAR-Fusion file (fusion-starfusion.tsv.gz)
+```{r}
+# check that all biospecimen IDs are in the file
+biospecimen_ids <- readr::read_tsv(file.path(data_dir, "fusion-starfusion.tsv.gz")) %>%
+  dplyr::pull(tumor_id) %>%
+  unique()
+hist_ids <- hist_df %>%
+  dplyr::filter(experimental_strategy == "RNA-Seq" |
+                  experimental_strategy == "Targeted Sequencing",
+                !is.na(RNA_library), 
+                sample_type == "Tumor",
+                !cohort %in% c("DGD", "TCGA")) %>%
```

Comment:
```suggestion
  dplyr::filter(experimental_strategy == "RNA-Seq",
                !cohort %in% c("GTEx", "TCGA")) %>%
```
I think this would work?